src/wp-includes/update.php

```diff
@@ -158,7 +158,7 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 			sprintf(
 				/* translators: %s: support forums URL */
 				__( 'An unexpected error occurred. Something may be wrong with ClassicPress.net or this server&#8217;s configuration. If you continue to have problems, please report this issue in our <a href="%s">support forum</a>.' ),
-				'https://forums.classicpress.net/c/support/'
+				'https://forums.classicpress.net/c/supportc/support/'
```

Comment:
One last thing to fix in this PR: there are a couple of URLs that now say `/c/supportc/support/`.